multi-node/aws/pkg/config/templates/stack-template.json

```diff
@@ -256,7 +256,11 @@
           {
             "DeviceName": "/dev/xvda",
             "Ebs": {
-              "VolumeSize": "{{.ControllerRootVolumeSize}}"
+              "VolumeSize": "{{.ControllerRootVolumeSize}}",
+              {{if gt .ControllerRootVolumeIOPS 0}}
```

Comment:
@colhom 

> I would also like to see a unit test in the cluster pkg which ensures that the cloudformation resource definitions for the controller instance and worker launch configuration match what is specified in the config.
>
> The way that stack tags are currently tested at the cluster layer is a good example for what you should do here.

I'm not sure how the example you gave me maps to this particular change because the stack tags are sent in its own field in `CreateStackInput` tag (thus much easier to test, as you pointed out), whereas my changes are sent in `TemplateBody` field with the rest of the stack code.

> the integrity of this templating operation can be covered via unit tests in the cluster pkg.

The template already had lots of instructions such as the `{{if ...}}` block you mentioned but from what I saw in the code, there's currently no test to check the result of the rendered stack for these other `if`s.

If I understood you correctly, what you want is a test to validate the rendered stack template in both scenarios (controller/worker iops == 0 and controller/worker iops > 0).

Is that enough?